specs/core/0_beacon-chain.md

```diff
@@ -1741,32 +1795,66 @@ process_deposit(state, deposit)
 
 Verify that `len(block.body.voluntary_exits) <= MAX_VOLUNTARY_EXITS`.
 
-For each `exit` in `block.body.voluntary_exits`:
-
-* Let `validator = state.validator_registry[exit.validator_index]`.
-* Verify that `validator.exit_epoch > get_delayed_activation_exit_epoch(get_current_epoch(state))`.
-* Verify that `get_current_epoch(state) >= exit.epoch`.
-* Verify that `bls_verify(pubkey=validator.pubkey, message_hash=signed_root(exit, "signature"), signature=exit.signature, domain=get_domain(state.fork, exit.epoch, DOMAIN_EXIT))`.
-* Run `initiate_validator_exit(state, exit.validator_index)`.
+For each `exit` in `block.body.voluntary_exits`, run the following function:
+
+```python
+def process_exit(exit: VoluntaryExit, state: BeaconState):
```

Comment:
Prefer to re-order the parameters: `state` and then `exit` as most helper functions + fix type hinting:
```suggestion
def process_exit(state: BeaconState, exit: VoluntaryExit) -> None:
```

Reply:
It seems `exit` is a keyword in Python. What do you think about using `voluntary_exit`?